pkg/code-generator/codegen/templates/snapshot_emitter_test_template.go

```diff
@@ -67,11 +70,14 @@ var _ = Describe("{{ upper_camel .Project.ProjectConfig.Version }}Emitter", func
 		cfg, err = clientcmd.BuildConfigFromFlags("", kubeconfigPath)
 		Expect(err).NotTo(HaveOccurred())
 
+{{- if $needs_crd_client }}
+
 		cache := kuberc.NewKubeCache()
+{{- end}}
 
-{{- if $needs_kube_client }}
-var kube kubernetes.Interface
-{{- end }}
+{{- if $needs_standard_client }}
```

Comment:
Seems a bit weird to use `needs_standard_client` and `need_crd_client` here, but not refer to them below.